client/app/components/about-modal/about-modal.html

```diff
@@ -1,7 +1,5 @@
-<a href="#"  ng-click="vm.open()" >
-    <span translate>
-      About
-    </span>
+<a href="#"  ng-click="vm.open()" translate>
```

Comment:
@ZitaNemeckova it should look like this:

`
<a href="#"  ng-click="vm.open()">{{ 'About' | translate}}</a>
`